ru/announcements/2020-12-31-Rust-1.49.0.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: 'Rust 1.49.0: aarch64 и улучшения во фреймворке тестирования'
+author: The Rust Release Team
+release: 'true'
+---
+
+Команда Rust рада сообщить о выпуске новой версии, 1.49.0. Rust — это язык программирования, позволяющий каждому создавать надёжное и эффективное программное обеспечение.
+
+Если вы установили предыдущую версию Rust средствами `rustup`, то для обновления до версии 1.49.0 вам достаточно выполнить следующую команду:
+
+```console
+rustup update stable
+```
+
+Если у вас ещё не установлен `rustup`, вы можете [установить его] с соответствующей страницы нашего веб-сайта, а также посмотреть [подробные примечания к выпуску на GitHub].
+
+## Что вошло в стабильную версию 1.49.0
+
+В этом выпуске мы добавили новые таргеты и улучшили фреймворк тестирования. Смотрите [подробные примечания к выпуску](https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1490-2020-12-31), чтобы узнать о других изменениях, не представленных в данном анонсе.
+
+### 64-bit ARM Linux перешёл в Tier 1
+
+Компилятор Rust поддерживает [широкий спектр платформ], но команда Rust не может поддерживать все их на одном уровне. Для более явного указания уровня поддержки каждой платформы, мы используем многоуровневую систему:
+
+- Tier 3 - технически, платформы поддерживаются компилятором, но мы не проверяли собирается ли на них код или проходят ли тесты, и мы не предоставляем каких-либо бинарных артефактов, как часть наших релизов.
+- Платформы из Tier гарантированно собираются, но мы не запускаем для них тесты: предоставляемые артефакты могут быть не рабочими или быть с багами.
+- Для платформ из Tier 1 предоставляются наибольшие гарантии и мы запускаем все тесты для этих платформ для каждого изменения, влитого в компилятор. Также мы предоставляем для них собранные артефакты.
+
+Начиная с Rust 1.49.0 платформа `aarch64-unknown-linux-gnu` передвигается на уровень поддержики Tier 1, предоставляя наши наибольшие гарантии для пользователей Linux, запущенных на 64-bit ARM системах! Мы ожидаем, что данное изменение принесёт пользу для всех: от встраиваемых систем до обычных компьютеров и серверов.
+
+Это важный этап для нашего проекта, с тех пор, как первая не-x86 платформа перешла на Tier 1: мы надеемся, этот шаг откроет путь для большего количества платформ, которые смогу достигнуть нашего высшего уровня поддержки.
```

Comment:
```suggestion
Это важный этап для нашего проекта, с тех пор, как первая не-x86 платформа перешла на Tier 1: мы надеемся, этот шаг откроет путь для большего количества платформ, которые смогу достичь нашего высшего уровня поддержки.
```

Reply:
> с тех пор

Тут другой "since" же. Предлагаю что-то в духе:

"Это важный этап для нашего проекта, поскольку это первая не-x86 платформа, получившая Tier 1 поддержку:"